### PR TITLE
update site config data

### PR DESCRIPTION
**Pull request by 112002**
update docs
_one small article error_
**//the license**

### CODE REVIEW

**Comment by pombredanne** [comment]
Thank!
Do you mind to add a DCO signoff?

**Comment by 112002** [comment]
> Thank!
> Do you mind to add a DCO signoff?

Sure! It will be great.

**Comment by pombredanne** [comment]
>Sure! It will be great.

We love the DCO Signoffs when they are great!

Can you join us on https://gitter.im/aboutcode-org/discuss to discuss your proposed changes?

File: about.html [hunk 1]
```diff
@@ -15,6 +15,7 @@ <h4 id="version"></h4>
         </div>
     </div>
   </body>
+  //here we will write the text file
```
Comment (pombredanne) [inline about.html:18]: What is this comment about?

Comment (112002) [reply about.html:18]: That's for the better understanding of users as we used text/javascript there in the next line

**Comment by steven-esser** [comment]
Closing due to stagnation.

**Pull request closed without merging**